.github/workflows/scripts/countline.py

```
@@ -315,24 +316,18 @@ def main():
             errors_found = True
             file_count += 1
             if file_count == 1:
-                print(
-                    """
-LINE COUNT ERROR: Files with excessive lines of code have been found\n"""
-                )
+                print("""
+LINE COUNT ERROR: Files with excessive lines of code have been found\n""")
 
             print("  Line count: {:>5} File: {}".format(line_count, filepath))
```

Ruff (0.14.13)

Use f-string instead of `format` call

Convert to f-string

(UP032)